socs/agents/lakeshore372/agent.py

```diff
@@ -1305,7 +1305,7 @@ def get_still_output(self, session, params=None):
 
         return True, "Current still output is {}".format(still_output)
 
-    @ocs_agent.param('configfile', type=str)
+    @ocs_agent.param('configfile', type=str, default=None)
```

Comment:
Also update the method signature override in the docstring to be: `input_configfile(configfile=None)`